config/mtl.toml

```diff
@@ -288,7 +288,7 @@ count = 15
 	instance_count = "8"
 	domain_types = "0"
 	load_type = "0"
-	module_type = "8"
+	module_type = "12"
```

Comment:
These values are interpreted as hex, hence MicSelect module_type should be '0xC'

Reply:
please check https://github.com/thesofproject/rimage/pull/139